### ISSUE TITLE
[Sourcing] [Accessibility] Popover is not opening with the enter and space keys, when in-focus inside the table cell.

### ISSUE THREAD

**Issue by Ankit0025-cmd** [SAP/fundamental-ngx#9668]
#### Is this a bug, enhancement, or feature request?
Bug
#### Briefly describe your proposal.
Popover should open when user presses 'Enter'/'Space' when the popover button is in focus.
#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
 "@fundamental-ngx/platform": "0.40.0-rc.15"
 "@angular/core": "15.2.0"
#### If this is a bug, please provide steps for reproducing it.
Just add the fd-popover with the trigger inside the table cell of an fdp-table and try to open the popover using the 'Enter' and 'Space' when the popover button is in focus. The popover opens when the button is everywhere but inside that fdp-table-cell.
#### Please provide relevant source code if applicable.
https://stackblitz.com/edit/angular-ojgcqx?file=package.json

**Comment by fkolar** [comment]
OK found the cause, the event is eaten by this `keydown.enter h`andler inside the datatable


```html
<td 
                            (keydown.enter)="_isTreeRowFirstCell(colIdx, row) && _toggleGroupRow(row)"
                        >
```

**Comment by fkolar** [comment]
Fix is ready, but waiting for other PRs to be merged.

**Comment by Ankit0025-cmd** [comment]
Issue seems not to be solved yet. Attaching the stackblitz link: https://stackblitz.com/edit/angular-ojgcqx?file=package.json 
I have tried with "0.40.2-rc.12" as well as "0.40.2-rc.12" but it's not responding with the space key.

**Comment by Ankit0025-cmd** [comment]
Hi @fkolar , any updates on this issue?

**Comment by fkolar** [comment]
> Hi @fkolar , any updates on this issue?

Checking..

**Comment by Ankit0025-cmd** [comment]
Hi @fkolar , 
The fix is working as expected in the 0.40.2-rc.19 version of the fundamental-ngx.
Now we are able to open the popover inside the table cell using keyboard(both 'enter' and 'space' keys).
Thanks
